### PR TITLE
[Misc] improve logits processors logging message

### PR DESCRIPTION
**Pull request by aw632**
Just a small improvement to the logits processors logging message so that the offending `token_id` is included in the error message.

Also removes an extraneous commented-out-line in `test_serving_chat.py`.

### CODE REVIEW

**Review by DarkLight1337 — APPROVED** [review]
Thanks for the QoL improvement!
